cogs/resourcecog.py

```diff
@@ -0,0 +1,169 @@
+from discord.ext import commands
+import discord
+import json
+from discord.utils import get
+
+
+def jsonsave(cfg):
+    with open("data/resources.json", "w") as f:
+        json.dump(cfg, f)
+
+
+def jsonopen():
```

Comment:
Since this function isn't just for opening jsons, and is actually specifically for loading the resource file, consider changing its name to be more reflective of its functionality.